cbw_api_toolbox/cbw_api.py

```diff
@@ -106,3 +106,21 @@ def remote_accesses(self):
         """GET request to /api/v2/remote_accesses to get all servers"""
         response = self._request("GET", [ROUTE_REMOTE_ACCESSES])
         return CBWParser().parse_response(CBWRemoteAccess, response)
+
+    def update_remote_access(self, remote_id, info):
+        """PATCH request to /api/v2/remote_accesses to update a specific server"""
```

Comment:
PATCH request to /api/v2/remote_accesses/{remote_id} to update a remote access

Reply:
:)